src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,88 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange:(value: number) => void;
+};
+
+export const Pagination: React.FC <Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesCount = Math.ceil(total / perPage);
+  const pages = getNumbers(1, pagesCount);
+  const firstPage = currentPage === 1;
+  const lastPage = currentPage === pagesCount;
+
+  const setPrevPage = () => {
+    if (!firstPage) {
+      onPageChange(currentPage - 1);
+    }
+  };
+
+  const setNextPage = () => {
```

Comment:
```suggestion
  const handleNextPage = () => {
```
pls see [naming convention](https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364)